src/core/sync/mutex.d

```diff
@@ -378,6 +378,7 @@ unittest
     // not run on Android.
     version (CRuntime_Bionic) {} else
     version (CRuntime_Musl) {} else
+    version (DragonFlyBSD) {} else               // FIXME : Will have to write a test program to see what is going on here.
```

Comment:
Still need to test this?

Reply:
Tested / Checked with dragonflybsd developers / Will not be corrected
FIXME: Comment removed
Comment above the test extended/updated.